<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VISAI00215L" failures="1" tests="9" name="API Tests - MeasurementImagebyID" time="17.288" errors="0" timestamp="2023-09-08T19:36:30 IST">
  <testcase classname="v1.Get.api.MeasurementImagebyID" name="idByImage" time="3.431"/>
  <testcase classname="v1.Get.api.MeasurementImagebyID" name="NoImageFound" time="1.383"/>
  <testcase classname="v1.Get.api.MeasurementImagebyID" name="Unauthorized" time="1.282"/>
  <testcase classname="v2.Get.api.MeasurementImagebyID" name="VeridyTheIdByImage" time="2.771"/>
  <testcase classname="v2.Get.api.MeasurementImagebyID" name="testNoImageFound" time="1.429">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at credentails.CommonMethods.imageIdAssert(CommonMethods.java:164)
at v2.Get.api.MeasurementImagebyID.testNoImageFound(MeasurementImagebyID.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testNoImageFound -->
  <testcase classname="v2.Get.api.MeasurementImagebyID" name="Unauthorized" time="1.22"/>
  <testcase classname="v3.Get.api.MeasurementImagebyID" name="idByImage" time="2.771"/>
  <testcase classname="v3.Get.api.MeasurementImagebyID" name="testNoImageFound" time="1.36"/>
  <testcase classname="v3.Get.api.MeasurementImagebyID" name="Unauthorized" time="1.63"/>
</testsuite> <!-- API Tests - MeasurementImagebyID -->
